snippets/javascript/statements.md

```diff
@@ -0,0 +1,50 @@
+# JS Statements
+
+## If statement
+
+```js
+const isMailSent = true;
+
+if (isMailSent) {
+  console.log("Mail sent to recipient");
+}
+```
+
+<!-->An if statement accepts an expression with a set of parentheses:
+
+If the expression evaluates to a truthy value, then the code within its code
+body executes. If the expression evaluates to a falsy value, its code body will
+not execute.<!-->
+
+## Else Statement
+
+```js
+const isTaskCompleted = false;
+
+if (isTaskCompleted) {
+  console.log("Task completed");
+} else {
+  console.log("Task incomplete");
+}
+```
+
+<!-->An else block can be added to an if block or series of if-else if blocks. The else block will be executed only if the if condition fails.<!-->
```

Comment:
Fails is not the right word here. Failing means something else in programming (throwing an error). I would reword to `if the if condition is falsy`